provisioning/service/samples/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "azure-iot-provisioning-samples",
+  "version": "0.0.1",
+  "description": "Azure IoT SDK - Provisioning Service Client samples",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "main": "iotprovisioningservice.js",
+  "typings": "iotprovisioningservice.d.ts",
```

Comment:
probably not the right values

Reply:
Just get rid of the main and the typings?

---
In reply to: [150136067](https://github.com/Azure/azure-iot-sdk-node/pull/142#discussion_r150136067) [](ancestors = 150136067)